README.md

```diff
@@ -1,7 +1,7 @@
 This is a place to practice git. Your first exercise is to add your name to this list.
-To complete this exercise you need to:
+To complete this exercise ou need to:
 
-- **Learn to use git**:
+- **Learn to use git in a comprehensive manner**:
```

Comment:
These changes are related to adding your name in the list. If you want to suggest changes to the guidelines, then open a separate pull request.